cmd2/utils.py

```diff
@@ -13,7 +13,7 @@
 import threading
 import unicodedata
 from enum import Enum
-from typing import Any, Callable, Dict, Iterable, List, Optional, TextIO, Type, Union
+from typing import Any, Callable, Dict, Iterable, List, Optional, TextIO, Union
```

Comment:
It would be extremely helpful if you can start documenting all the crap we can and should rip out once we deprecate support for 3.5 in the open issue for 3.5 deprecation.